core/validation/failures/user.py

```diff
@@ -376,14 +378,14 @@ def normalise_risk_section_name(section):
     null_messages = [msgs.BLANK, msgs.BLANK_ZERO, msgs.BLANK_PSI, msgs.BLANK_UNIT_OF_MEASUREMENT]
 
     null_failures = [
-        (sheet, section, cell_index.strip(), message)
-        for sheet, section, cell_index, message in errors
+        (sheet, section, cell_index.strip(), message, _)
+        for sheet, section, cell_index, message, _ in errors
```

Comment:
We should use a variable name here (and throughout this function). Good practice to use `_` if you don't use the variable that's unpacked